src/components/Menu.js

```diff
@@ -242,6 +249,7 @@ export class Menu extends Component<MenuProps, MenuState> {
   state = {
     maxHeight: this.props.maxMenuHeight,
     placement: null,
+    theme: null,  // TODO: is this okay
```

Comment:
@akx I think I'm missing something; why would the `theme` be in `state`?

Also, what's the TODO in reference to?

Reply:
Yeah, no idea either... Wonder what I'd been thinking there.